auth/package.json

```diff
@@ -1,7 +1,7 @@
 {
     "name": "@microsoft/vscode-azext-azureauth",
     "author": "Microsoft Corporation",
-    "version": "4.1.0",
+    "version": "4.1.1",
```

Comment:
Can you update the version in package-lock.json too?

Reply:
🙈